packages/deploy-tooling/test/unit/base/validate.test.ts

```diff
@@ -480,7 +480,7 @@ describe('deploy-test validation', () => {
             }
         );
     });
-    describe('Validate error does not show full stack trace', () => {
+    describe.only('Validate error does not show full stack trace', () => {
```

Comment:
```suggestion
    describe('Validate error does not show full stack trace', () => {
```

Reply:
Thank you Austin, addressed in [c824d87](https://github.com/SAP/open-ux-tools/pull/1809/commits/c824d875528b1c9916749f8bbe0982c31b27a998)